make/common/native/Flags.gmk

```diff
@@ -234,6 +234,9 @@ define SetupLinkerFlags
     ifeq ($(call isTargetOs, macosx), true)
       $1_EXTRA_LDFLAGS += -Wl,-object_path_lto,$$($1_OBJECT_DIR)/$$($1_NAME)_lto_helper.o
     endif
+
+    # for lto, we have to use the debug prefix mapping flags to get rid of unwanted paths
+    # $1_EXTRA_LDFLAGS += $$(DEBUG_PREFIX_CFLAGS)
```

Comment:
This looks redundant. It should be enough to add the flags to LDFLAGS_LTO in configure.